src/qt/sendcoinsdialog.cpp

```diff
@@ -369,6 +370,8 @@ void SendCoinsDialog::on_sendButton_clicked()
         accept();
         CoinControlDialog::coinControl()->UnSelectAll();
         coinControlUpdateLabels();
+        const uint256 txid = currentTransaction.getTransaction()->GetHash();
+        Q_EMIT coinsSent(QString::fromStdString(txid.ToString()));
```

Comment:
Meh on using strings for internal uint256 transport.
Why not uint256? Could also register the type if not yet supported.

Reply:
I tried passing the uint256 directly and got a bloodbath of compiler errors. How do I go about registering the type?

Reply:
With `Q_DECLARE_METATYPE`, see https://github.com/bitcoin/bitcoin/blob/dcfe218626b05204e9fbc95ba5d95ca0eb72ec9b/src/qt/bitcoin.cpp#L80-L82